### PR TITLE
Support for received votes - Closes #189

### LINKED ISSUE #189: Support for received voters
### Description

Make Lisk Service support sent votes. 

- Update the blockchain compatibility layer to work with SDKv4
- Create blockchain data store for voters
- Update the extended attributes (meta)
- Move `/api/v1/voters` to the version 1 compatibility API
- Update the URL in the official api `/api/v1/voters` -> `/api/v1/votes_received`
- Add support for the `amount` property

### Motivation

- Lisk UI frontend projects require this path to make the client possible to retrieve voting information

### Acceptance Criteria

- SDK v2 Lisk Core works
- SDK v3 Lisk Core works
- SDK v4 Lisk Core works
- `/api/v1/votes_received` is available
- `/api/v1/voters` is available

### PR DESCRIPTION
**Pull request by nagdahimanshu**
### What was the problem?
This PR resolves #189 

### How was it solved?
- [x] Update the blockchain compatibility layer to work with SDKv4
- [x] Create blockchain data store for voters
- [x] Update the extended attributes (meta)
- [x] Move /api/v1/voters to the version 1 compatibility API
- [x] Update the URL in the official api /api/v1/voters -> /api/v1/votes_received
- [x] Add support for the amount property

### How was it tested?
Locally + Jenkins

### CODE REVIEW

**Review by sameersubudhi — APPROVED** [review]
LGTM! 👍

File: services/gateway/apis/http-version1/methods/voters.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ const { transformParams, response } = require('../swagger/utils');
 
 module.exports = {
 	version: '2.0',
-	swaggerApiPath: '/voters',
+	swaggerApiPath: '/votes_received',
 	rpcMethod: 'get.voters',
```
Comment (MichalTuleja) [inline services/gateway/apis/http-version1/methods/voters.js:23]: The RPC method name should be updated as well. Can you please double-check `votes_sent` as well?

**Review by MichalTuleja — CHANGES_REQUESTED** [review]
Please fix the RPC name and we're good to go 👍

File: services/gateway/apis/http-version1/methods/voters.js (hunk 1 above)
Comment (nagdahimanshu) [reply services/gateway/apis/http-version1/methods/voters.js]: Updated in both votes and voters.

**Review by MichalTuleja — APPROVED** [review]

**Pull request merged**